src/containers/shared/MetaParser.tsx

```diff
@@ -0,0 +1,111 @@
+import { XRP_BASE } from './transactionUtils'
```

Comment:
nit: this file should be `metadataParser`(lowercase because it isn't a React component).

Reply:
will do